lib/node_modules/@stdlib/math/base/special/sincospi/README.md

```diff
@@ -70,7 +70,7 @@ for ( i = 0; i < x.length; i++ ) {
 
 [@stdlib/math/base/special/cos]: https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/base/special/cos
 
-[@stdlib/math/constants/pi]: https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/constants/pi
+[@stdlib/math/constants/pi]: https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/constants/float64-pi
```

Comment:
This identifier is also wrong. Should be `@stdlib/math/constants/float64-pi`.

Reply:
Ah, is that used for rewriting links? Even if not, it'd only make sense. At any rate, done.

Reply:
That is correct. :) Package names serve as unique identifiers which we can leverage in definitions to rewrite links. We exploit this in various `remark` plugins when we target both the repository hosted on GitHub and the website in order to resolve internal project links.

In fact, the id is actual more important, in this case, than the URL itself. We can automatically fix any URLs via a build process. Bad identifiers not so much.